.travis.yml

```diff
@@ -19,6 +24,7 @@ deploy:
     skip_cleanup: true
     script: bash $TRAVIS_BUILD_DIR/.travis.d/deploy_docs.sh
     on:
+        repo: vitasdk/vita-headers
```

Comment:
yes, if someone want to deploy their repo, just need remove that line and push empty gh-pages branch.

[this](https://travis-ci.org/d3m3vilurr/vita-headers/builds/177312256) is deploy test in local repo
[this](https://travis-ci.org/d3m3vilurr/vita-headers/builds/177312697) is prevent deploy result.